src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.ts

```diff
@@ -144,4 +146,40 @@ export class ContentTypeFieldsPropertiesFormComponent extends BaseComponent impl
             this.fieldPropertyService.getOrder(property1) - this.fieldPropertyService.getOrder(proeprty2));
 
     }
+
+    private handleAutoCheckValues(): void {
+        if (this.form.get('searchable')) {
+            this.form.get('searchable').valueChanges.subscribe(res => this.setIndexedValueChecked(res));
+        }
+        if (this.form.get('listed')) {
+            this.form.get('listed').valueChanges.subscribe(res => this.setIndexedValueChecked(res));
+        }
+        if (this.form.get('unique')) {
+            this.form.get('unique').valueChanges.subscribe(res => this.handleUniqueValuesChecked(res));
+        }
+    }
+
+    private setIndexedValueChecked(propertyValue: boolean): void {
+        if (this.form.get('indexed')) {
+            this.form.get('indexed').setValue(propertyValue);
+            this.handleDisabledIndexed(propertyValue);
```

Comment:
Here you can do directly:

```this.form.get('indexed').disable(propertyValue)``` <-- assuming propertyValue its true/false

Reply:
I'm not able to pass propertyValue or boolean to .disable(). I get this warning: 
[ts] Argument of type 'boolean' is not assignable to parameter of type '{ onlySelf?: boolean; emitEvent?: boolean; }'. 
(parameter) propertyValue: boolean

Reply:
Meh... I thought we can pass the value to it.